imgix/urlbuilder.py

```diff
@@ -46,17 +46,24 @@ def __init__(
             use_https=True,
             sign_key=None,
             shard_strategy=SHARD_STRATEGY_CRC,
-            sign_with_library_version=True):
+            sign_with_library_version=None,
+            include_library_param=True):
+
+        if sign_with_library_version is not None:
+            warnings.warn('`sign_with_library_version` has been deprecated.' +
```

Comment:
Can we be a bit more specific here? "... has been deprecated and will be removed in the next major version."